flink-streaming-connectors/flink-connector-kafka/src/main/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaConsumer.java

```diff
@@ -632,6 +632,7 @@ public void run() {
 						//  ------------  commit current offsets ----------------
 
 						// create copy of current offsets
+						//noinspection unchecked
```

Comment:
Is this this is only suppressing IntellIJ warnings or is this meant to be a replacement for the `@SuppressWarnings` annotation?